test.sh

```diff
@@ -0,0 +1,3 @@
+#!/bin/bash
+sudo docker build --force-rm -t pytrac/debian:stable .
+sudo docker run -rm -t -i  -v $PWD:/home/pytrac/data pytrac/debian:stable ./venv/bin/python ./setup.py test
```

Comment:
@mlafeldt giving full docker access to normal linux users is a security issue as it essentially gives them root.